ext/search_kit/Civi/Api4/Action/SearchDisplay/GetSearchTasks.php

```diff
@@ -41,7 +41,8 @@ public function _run(\Civi\Api4\Generic\Result $result) {
         'icon' => 'fa-file-excel-o',
         'crmPopup' => [
           'path' => "'civicrm/export/standalone'",
-          'query' => "{reset: 1, entity: '{$entity['name']}', id: ids.join(',')}",
+          'query' => "{reset: 1, entity: '{$entity['name']}'}",
+          'data' => "{id: ids.join(',')}",
```

Comment:
@eileenmcnaughton this PR addresses all actions but export - we need to add qfKey here but need a function that returns the quickform export class name given an entity name. Can't find one...

Reply:
Nevermind I figured it out.